src/test/groovy/com/ofg/uptodate/reporting/NewVersionLoggerSpec.groovy

```diff
@@ -0,0 +1,55 @@
+package com.ofg.uptodate.reporting
+
+import com.ofg.uptodate.LoggerProxy
+import com.ofg.uptodate.dependency.Dependency
+import spock.lang.Specification
+
+import static com.ofg.uptodate.reporting.NewVersionLogger.NEW_VERSIONS_AVAILABLE
+import static com.ofg.uptodate.reporting.NewVersionLogger.NEW_VERSIONS_MESSAGE_HEADER
+import static com.ofg.uptodate.reporting.NewVersionLogger.NO_NEW_VERSIONS_AVAILABLE
+import static com.ofg.uptodate.reporting.NewVersionLogger.NO_NEW_VERSIONS_MESSAGE
+
+class NewVersionLoggerSpec extends Specification {
+
+    static final Dependency HIBERNATE = new Dependency('org.hibernate', 'hibernate-core', '4.3.6.Final')
+    static final Dependency JUNIT = new Dependency('junit', 'junit', '4.11')
+    static final String PROJECT_NAME = 'ProjectName'
+
+    LoggerProxy logger = Mock()
+
+    def 'should report updates in lexical order'() {
+        given:
+            NewVersionLogger reporter = new NewVersionLogger(logger, PROJECT_NAME, false)
+            List<Dependency> updates = [HIBERNATE, JUNIT]
+        when:
+            reporter.reportUpdates(updates as Set)
+        then:
+            1 * logger.lifecycle(_, "$NEW_VERSIONS_MESSAGE_HEADER$JUNIT\n$HIBERNATE")
```

Comment:
I don't know if in this test (here is some logic tested), but maybe it would be useful to have a tests that verifies that output is in expected format (an acceptance tests)?

Something like:
```
New versions available:
org.hibernate:hibernate-core:4.3.6.Final
junit:junit:4.11
```

To prevent situation that modified `toString` makes the output unreadable. Just an idea, WDYT?